server/playData.js

```diff
@@ -64,10 +61,10 @@ function parseScript(playPath, title) {
   scriptData.fullPlay = fullPlay;
   scriptData.characterObjs = characterObjs;
 
-  return scriptData;
+  playData[scriptData.title] = scriptData;
 }
 
-const test = parseScript(testScriptPath, 'test');
-const twelfthNight = parseScript(twelfthNightPath, 'twelfthNight');
+const TestPlay = parseScript(testScriptPath, 'Test Play');
+const TwelfthNight = parseScript(twelfthNightPath, 'Twelfth Night');
```

Comment:
Also, it can sometimes be clearer to keep with your existing format of function taking in data and returning new data. _Then_ the user of that function can decide what to do with it. Having the data saved to playData as a side effect of the function can make code harder to follow or make other developers have to read more of the code to understand what's happening to the data passed to your function.

Reply:
Excellent point. I'm in the middle of refactoring this in my upload branch so I will do this restructuring over there.